app/controllers/likes_controller.rb

```diff
@@ -1,7 +1,27 @@
 class LikesController < ApplicationController
   def create
+    @bookmark = Bookmark.find(params[:bookmark_id])
```

Comment:
Minimize variable scope when possible. `@bookmark` can be a local variable `bookmark` as we don't use it in a view rendered by this controller. It would only need to be an instance variable `@bookmark` if we rendered a template in this controller that needed a bookmark object.

Same goes for the `destroy` action in this controller.